google/resource_container_cluster.go

```diff
@@ -370,6 +370,13 @@ func resourceContainerCluster() *schema.Resource {
 				Description: `Enable Shielded Nodes features on all nodes in this cluster. Defaults to false.`,
 			},
 
+			"enable_autopilot": {
+				Type:        schema.TypeBool,
+				Optional:    true,
```

Comment:
It does not appear that the autopilot flag is updatable, so we should add `ForceNew: true` here to force recreate the cluster on the field update.